### PR TITLE
Create app Poll

### PR DESCRIPTION
**Pull request by aliskycloud**
Milad, Plz chk task
Sepehr, Check the poll_info.html file in the templates directory
@millad90s 

┆Issue is synchronized with this [Trello card](https://trello.com/c/sH3l0ZDK) by [Unito](https://www.unito.io)

### CODE REVIEW

File: firstapp/firstapp/settings.py [hunk 1]
```diff
@@ -39,6 +39,9 @@
     'django.contrib.staticfiles',
     'home',
     'mbk',
+    'polls',
+
```
Comment (millad90s) [inline firstapp/firstapp/settings.py:43]: empty lines !!

File: firstapp/polls/models.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+# Create your models here.
+class Question(models.Model):
+    question = models.TextField()
+    start_date = models.DateTimeField(auto_now_add=True)
+    end_date = models.DateTimeField()
+
```
Comment (millad90s) [inline firstapp/polls/models.py:9]: search about __str__ in python !

File: firstapp/polls/views.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from .models import Question
+
+
+def say_hello(request):
+    return HttpResponse('Hello poll')
```
Comment (millad90s) [inline firstapp/polls/views.py:7]: do we need this function !

File: firstapp/polls/views.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+
+def template_poll(request):
+    return render(request, 'poll_info.html')
```
Comment (millad90s) [inline firstapp/polls/views.py:11]: what is it for ?

**Review by millad90s — CHANGES_REQUESTED** [review]

**Comment by aliskycloud** [comment]
@millad90s 
@Sepii84 
I pushed again after the corrections

**Comment by Sepii84** [comment]
The Html file is all good as far as I can see

**Pull request merged**